packages/lesswrong/components/tagging/TagFilterSettings.tsx

```diff
@@ -65,12 +65,14 @@ const TagFilterSettings = ({
   setPersonalBlogFilter,
   setTagFilter,
   removeTagFilter,
+  disableFlexWrapEndGrow = false,
```

Comment:
I would remove the negation in both the name and default value (i.e. `flexWrapEndGrow = true`), which is simpler to understand, and then you can get rid of the negation on L130